control-plane/subcommand/server-acl-init/command.go

```diff
@@ -319,12 +323,7 @@ func (c *Command) Run(args []string) int {
 
 	var bootstrapToken string
 
-	if c.flagBootstrapTokenFile != "" {
-		// If bootstrap token is provided, we skip server bootstrapping and use
-		// the provided token to create policies and tokens for the rest of the components.
-		c.log.Info("Bootstrap token is provided so skipping Consul server ACL bootstrapping")
-		bootstrapToken = providedBootstrapToken
-	} else if c.flagACLReplicationTokenFile != "" && !c.flagCreateACLReplicationToken {
```

Comment:
previously, we'd skip server bootstrapping (including setting server agent tokens) because we assumed you're running with external servers. We no longer want to do when the bootstrap token is stored in vault.